schema/stateTransition/base.json

```diff
@@ -8,11 +8,30 @@
     "type": {
       "type": "number",
       "enum": [1, 2]
+    },
+    "signaturePublicKeyId": {
+      "anyOf": [
```

Comment:
We don't need `anyOf` here. Just `type: [ "string", "null" ]`

Reply:
I think it is null or integer

Reply:
Yeah, sure! Sorry.